### ISSUE TITLE
Can you mention the hyper-parameters used to reproduce baseline(random) cifar10 results?

### ISSUE THREAD

**Issue by gauravfotedar** [sinhasam/vaal#13]
Hyper-parameters like batch_size, number of epochs and learning rate
